java/src/jmri/Block.java

```diff
@@ -135,7 +135,7 @@ public Block(String systemName, String userName) {
     static final public int TIGHT = 0x02;
     static final public int SEVERE = 0x04;
 
-    // this should only be used for debugging…
+    // this should only be used for debugging
```

Comment:
My last PR (which has been merged now) should have all LayoutEditor and associated files purged of ellipses…